src/StackManager.js

```diff
@@ -72,21 +76,26 @@ export default class StackManager {
     const file = dockerComposePath(this.getProjectPath());
     const command = clean(args).join(' ');
 
-    return exec(`docker-compose --file ${file} ${command}`);
+    return exec(`docker-compose -f ${file} -p ${this.getProjectName()} ${command}`);
```

Comment:
we should make the signature of `exec` similar to `spawn`: command, args and options